### PR TITLE
python310Packages.certbot-dns-inwx: init at 2.1.3

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/certbot-dns-inwx/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    sha256 = "sha256-yAgualY4J92pJ8PIkICg8w0eYHmT5L3qAUOCW/cAitw=";
+  };
+
+  propagatedBuildInputs = [
+    acme
+    certbot
+  ];
+
+  # Doesn't have any tests
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/certbot-dns-inwx/default.nix:23]: Can we add a pythonImportsCheck?

**Comment by mweinelt** [comment]
I don't quite understand why anyone would want to use certbot when we have a proper nixos module for ACME backed by lego, that also supports INWX, See https://go-acme.github.io/lego/dns/inwx/.

**Comment by onny** [comment]
> I don't quite understand why anyone would want to use certbot when we have a proper nixos module for ACME backed by lego, that also supports INWX, See https://go-acme.github.io/lego/dns/inwx/.

Thanks for the hint. That's something I missed, good to know. Nevertheless `certbot` is still actively maintained and it would be nice to support it too (thinking about enhancing the wiki article further and maybe also add a module for it).

**Comment by SuperSandro2000** [comment]
> Nevertheless `certbot` is still actively maintained and it would be nice to support it too

But it would add more maintenance work for us and I think that outweighs nice to have.

**Comment by onny** [comment]
@SuperSandro2000 Added the `pytestCheckHook`, thanks for the note!

Would be really happy if we could merge this :)

**Comment by happysalada** [comment]
I propose that we try to include this.
Onny wants to maintain it. If it turns out to be too much of a burden for other people, we can always decide to remove it.
(let me know if you disagree though).

**Comment by SuperSandro2000** [comment]
> If it turns out to be too much of a burden for other people, we can always decide to remove it.

The problem is not that we have a single program or feature that requires to much maintenance itself but a very many ones that require just a little bit of time which in total is also a lot of time.

**Comment by happysalada** [comment]
I agree with you here.
I feel if Onny wants to maintain this, we should let him try to do it and assume it will work out.
I think it's hard to tell people what to maintain, you can only let people maintain what they want.
I know python packages are tricky since sometimes an update can be blocked by packages, let's definitely revisit this conversation if it turns out this particular package is a pain.
We need more contributors (I think we will both agree on this), hopefully by making existing contributors a little happier they can attract more contributors (I have no scientific proof for this).

Let me take this opportunity to thank you for all the work you are doing for nixpkgs, I can't imagine nixpkgs without your contributions!

**Pull request merged**